docs/modules/ROOT/pages/release-notes/release-notes-34-0.adoc

```diff
@@ -0,0 +1,32 @@
+# Release Notes - Payara Platform Enterprise 5.34.0
+
+### Supported APIs and Applications
+* Jakarta EE 8
+* Java EE 8 Applications
+* MicroProfile 4.1
+
+
+### Improvements
+[FISH-5745] Allow 'extraParameters' to be Specified via MP Config
```

Comment:
Preview doesn't render this correctly. See https://deploy-preview-221--docs-payara-enterprise.netlify.app/docs/release-notes/release-notes-34-0.html

An empty line is required between each item.